test/test_common/environment.cc

```diff
@@ -71,6 +61,28 @@ char** argv_;
 
 } // namespace
 
+void TestEnvironment::createPath(const std::string& path) {
+#ifdef __APPLE__
+  // No support in Clang OS X libc++ today for std::filesystem.
+  RELEASE_ASSERT(::system(("mkdir -p " + path).c_str()) == 0, "");
+#else
+  // We don't want to rely on mkdir etc. if we can avoid it, since it might not
+  // exist in some environments such as ClusterFuzz.
+  std::experimental::filesystem::create_directories(std::experimental::filesystem::path(path));
+#endif
+}
+
+void TestEnvironment::createParentPath(const std::string& path) {
+#ifdef __APPLE__
+  // No support in Clang OS X libc++ today for std::filesystem.
+  RELEASE_ASSERT(::system(("mkdir -p $(dirname " + path + ")").c_str()) == 0, "");
+#else
+  // We don't want to rely on mkdir etc. if we can avoid it, since it might not
+  // exist in some environments such as ClusterFuzz.
+  std::experimental::filesystem::create_directories(
```

Comment:
Do you want to RELEASE_ASSERT on failure here?

Reply:
This will throw an exception on failure, so we should hopefully see something interesting :)